package.json

```diff
@@ -59,6 +59,7 @@
     "husky": "^7.0.4",
     "lint-staged": "^12.3.7",
     "lockfile": "^1.0.4",
+    "npkill": "^0.12.2",
```

Comment:
Only curious why we need this dependency

Reply:
common dev tool for resetting node_modules